src/main/resources/hudson/plugins/ec2/SlaveTemplate/help-connectBySSHProcess.html

```diff
@@ -0,0 +1,34 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-, Kohsuke Kawaguchi, Sun Microsystems, Inc., and a number of other of contributors
```

Comment:
@tfennelly You added the file, you're the author, so your (or your employer's) copyright goes here. I'm not sure 31bc2f5730dcf294d6665973f36a03c4fd39fd5c was correct.